### ISSUE TITLE
Bulk dark-mode and/or high-contrast bug report

### ISSUE THREAD

**Issue by allejo** [StackExchange/Stacks#766]
- [x] **Activity indicator in high-contrast mode has white text**

![image](https://user-images.githubusercontent.com/1246453/135511120-30196610-cdbe-4fc9-aaf5-c865be71ddb1.png)

- [x] **Button Group in dark-mode doesn't have much contrast for selected**

![image](https://user-images.githubusercontent.com/1246453/135511605-fe397dfe-8fe5-4412-9dd3-94ee368a9114.png)

- [x] **Checkbox and Radio icon is white on light blue in high-contrast**

![image](https://user-images.githubusercontent.com/1246453/135511751-a6e1d9b0-6292-41d2-9118-056aa29dc6ae.png)

![image](https://user-images.githubusercontent.com/1246453/135512294-6f7a0723-7f36-4f4c-9fd9-8a4229e7aee8.png)

- [ ] **Disabled buttons in Editor component in high-contrast mode are hard to see**

![image](https://user-images.githubusercontent.com/1246453/135512064-00a2b214-4983-4c72-b4f1-939ffde4c033.png)

- [x] **Link to heading buttons don't have any contrast on focus over hover in high-contrast mode**

![image](https://user-images.githubusercontent.com/1246453/135513163-e7375a3a-74de-4716-8a76-f084c3db7d5a.png)

vs dark-mode (which is good)

![image](https://user-images.githubusercontent.com/1246453/135513210-e1313df5-b5f3-4900-89aa-f91a1583079d.png)

- [x] **Uploader component doesn't have contrast on "file" in high-contrast mode (both dark and light); applies to multi-upload as well**

![image](https://user-images.githubusercontent.com/1246453/135513388-1729d78a-89db-4793-adab-950a369ea37d.png)

![image](https://user-images.githubusercontent.com/1246453/135513445-3462673d-d03a-4192-99fe-c25146e00fa5.png)

![image](https://user-images.githubusercontent.com/1246453/135513671-a97b2ac9-ad6b-4ac0-915b-3e493e434d71.png)

vs. non-high-contrast mode where there's a difference

![image](https://user-images.githubusercontent.com/1246453/135513517-77773408-0b0b-4053-83c5-3534514cf175.png)

![image](https://user-images.githubusercontent.com/1246453/135513641-35e4814c-3a20-4639-b6e8-bb5e74085631.png)

- [x] **Icon set on docs no visible in high-contrast mode**

![image](https://user-images.githubusercontent.com/1246453/135514001-37dee243-1bcb-43d0-a494-18b8cc4f100a.png)

vs

![image](https://user-images.githubusercontent.com/1246453/135514032-c29f4b49-7fd0-440c-98fb-a3dc675dc2d5.png)

- [x] **Buttons in dark-mode and high-contrast mode are inconsistent**

1. when a `.s-btn__filled` (also `.s-btn__primary`) is given focus in high-contrast mode, it's not visible nor is there an outline
   ![image](https://user-images.githubusercontent.com/1246453/135514329-33564f92-8d9f-440a-9db9-dafc2a6b28b2.png)
2. when a `.s-btn__filled` (also `.s-btn__primary`) is given focus in dark-mode, there isn't a border around like there is in light-mode
    ![image](https://user-images.githubusercontent.com/1246453/135514432-ab60ef1a-0489-4fd8-b2d7-bed689d2fa4f.png)
    ![image](https://user-images.githubusercontent.com/1246453/135514506-9ac72372-fac3-460a-b9f9-0fe8e57804df.png)

**Comment by dancormier** [comment]
Thanks so much for detailing these @allejo!

I have a PR that addresses these issues over at https://github.com/StackExchange/Stacks/pull/764. I can't seem to add you as a reviewer, but feel free to leave notes as you see fit here, there, or anywhere.

https://github.com/StackExchange/Stacks/pull/764 has fixes for everything you mentioned except the editor disabled button contrast issue (I added a new issue to the Stacks-Editor repo https://github.com/StackExchange/Stacks-Editor/issues/86).
